[2024-10-24T01:26:58.092+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_openbrewerydb_to_silver_gold.Silver scheduled__2023-12-02T08:00:00+00:00 [queued]>
[2024-10-24T01:26:58.113+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_openbrewerydb_to_silver_gold.Silver scheduled__2023-12-02T08:00:00+00:00 [queued]>
[2024-10-24T01:26:58.114+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2024-10-24T01:26:58.316+0000] {taskinstance.py:1380} INFO - Executing <Task(SparkSubmitOperator): Silver> on 2023-12-02 08:00:00+00:00
[2024-10-24T01:26:58.327+0000] {standard_task_runner.py:57} INFO - Started process 8858 to run task
[2024-10-24T01:26:58.340+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'api_openbrewerydb_to_silver_gold', 'Silver', 'scheduled__2023-12-02T08:00:00+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/api_openbrewerydb_to_silver.py', '--cfg-path', '/tmp/tmpzcbc4665']
[2024-10-24T01:26:58.367+0000] {standard_task_runner.py:85} INFO - Job 141: Subtask Silver
[2024-10-24T01:26:58.626+0000] {task_command.py:415} INFO - Running <TaskInstance: api_openbrewerydb_to_silver_gold.Silver scheduled__2023-12-02T08:00:00+00:00 [running]> on host de5108c6da1d
[2024-10-24T01:26:58.952+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rodrigo Araujo' AIRFLOW_CTX_DAG_ID='api_openbrewerydb_to_silver_gold' AIRFLOW_CTX_TASK_ID='Silver' AIRFLOW_CTX_EXECUTION_DATE='2023-12-02T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-02T08:00:00+00:00'
[2024-10-24T01:26:59.031+0000] {base.py:73} INFO - Using connection ID 'spark-conn' for task execution.
[2024-10-24T01:26:59.036+0000] {spark_submit.py:363} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --name arrow-spark --queue root.default jobs/silver/api_openbrewerydb.py
[2024-10-24T01:27:30.886+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:30 INFO SparkContext: Running Spark version 3.5.3
[2024-10-24T01:27:30.911+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:30 INFO SparkContext: OS info Linux, 5.15.0-124-generic, amd64
[2024-10-24T01:27:30.928+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:30 INFO SparkContext: Java version 11.0.25
[2024-10-24T01:27:31.711+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:31 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-10-24T01:27:32.963+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:32 INFO ResourceUtils: ==============================================================
[2024-10-24T01:27:32.965+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:32 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-10-24T01:27:32.976+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:32 INFO ResourceUtils: ==============================================================
[2024-10-24T01:27:32.979+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:32 INFO SparkContext: Submitted application: IngestaoSilver
[2024-10-24T01:27:33.392+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:33 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-10-24T01:27:33.611+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:33 INFO ResourceProfile: Limiting resource is cpu
[2024-10-24T01:27:33.614+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:33 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-10-24T01:27:34.225+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:34 INFO SecurityManager: Changing view acls to: ***
[2024-10-24T01:27:34.233+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:34 INFO SecurityManager: Changing modify acls to: ***
[2024-10-24T01:27:34.235+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:34 INFO SecurityManager: Changing view acls groups to:
[2024-10-24T01:27:34.237+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:34 INFO SecurityManager: Changing modify acls groups to:
[2024-10-24T01:27:34.239+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:34 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-10-24T01:27:37.378+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:37 INFO Utils: Successfully started service 'sparkDriver' on port 36309.
[2024-10-24T01:27:37.746+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:37 INFO SparkEnv: Registering MapOutputTracker
[2024-10-24T01:27:37.942+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:37 INFO SparkEnv: Registering BlockManagerMaster
[2024-10-24T01:27:38.157+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:38 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-10-24T01:27:38.160+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:38 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-10-24T01:27:38.231+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:38 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-10-24T01:27:38.441+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:38 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-3d7fd9df-1cad-4480-adb7-db117842ef5d
[2024-10-24T01:27:38.582+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:38 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-10-24T01:27:38.784+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:38 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-10-24T01:27:41.587+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:41 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-10-24T01:27:42.438+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-10-24T01:27:42.452+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2024-10-24T01:27:42.461+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2024-10-24T01:27:42.465+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2024-10-24T01:27:42.479+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2024-10-24T01:27:42.481+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2024-10-24T01:27:42.482+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2024-10-24T01:27:42.489+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4047. Attempting port 4048.
[2024-10-24T01:27:42.491+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 WARN Utils: Service 'SparkUI' could not bind on port 4048. Attempting port 4049.
[2024-10-24T01:27:42.632+0000] {spark_submit.py:524} INFO - 24/10/24 01:27:42 INFO Utils: Successfully started service 'SparkUI' on port 4049.
[2024-10-24T01:29:31.340+0000] {spark_submit.py:524} INFO - 24/10/24 01:29:31 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-24T01:29:56.191+0000] {spark_submit.py:524} INFO - 24/10/24 01:29:56 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-24T01:30:51.583+0000] {spark_submit.py:524} INFO - 24/10/24 01:30:38 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-24T01:30:52.726+0000] {spark_submit.py:524} INFO - 24/10/24 01:30:52 WARN TransportClientFactory: DNS resolution succeed for spark-master/172.22.0.3:7077 took 6563 ms
[2024-10-24T01:31:03.346+0000] {spark_submit.py:524} INFO - 24/10/24 01:31:03 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2024-10-24T01:31:10.580+0000] {spark_submit.py:524} INFO - 24/10/24 01:31:03 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2024-10-24T01:33:59.587+0000] {spark_submit.py:524} INFO - 24/10/24 01:33:59 WARN NettyRpcEnv: Ignored failure: java.io.IOException: Connecting to spark-master/172.22.0.3:7077 timed out (120000 ms)
[2024-10-24T01:33:59.626+0000] {spark_submit.py:524} INFO - 24/10/24 01:33:59 ERROR RpcOutboxMessage: Ask terminated before connecting successfully
[2024-10-24T01:33:59.684+0000] {spark_submit.py:524} INFO - 24/10/24 01:33:59 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark-master:7077
[2024-10-24T01:33:59.691+0000] {spark_submit.py:524} INFO - org.apache.spark.rpc.RpcTimeoutException: Futures timed out after [120 seconds]. This timeout is controlled by spark.rpc.lookupTimeout
[2024-10-24T01:33:59.692+0000] {spark_submit.py:524} INFO - at org.apache.spark.rpc.RpcTimeout.org$apache$spark$rpc$RpcTimeout$$createRpcTimeoutException(RpcTimeout.scala:47)
[2024-10-24T01:33:59.692+0000] {spark_submit.py:524} INFO - at org.apache.spark.rpc.RpcTimeout$$anonfun$addMessageIfTimeout$1.applyOrElse(RpcTimeout.scala:62)
[2024-10-24T01:33:59.692+0000] {spark_submit.py:524} INFO - at org.apache.spark.rpc.RpcTimeout$$anonfun$addMessageIfTimeout$1.applyOrElse(RpcTimeout.scala:58)
[2024-10-24T01:33:59.693+0000] {spark_submit.py:524} INFO - at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:38)
[2024-10-24T01:33:59.693+0000] {spark_submit.py:524} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:76)
[2024-10-24T01:33:59.693+0000] {spark_submit.py:524} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2024-10-24T01:33:59.693+0000] {spark_submit.py:524} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2024-10-24T01:33:59.694+0000] {spark_submit.py:524} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:108)
[2024-10-24T01:33:59.694+0000] {spark_submit.py:524} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2024-10-24T01:33:59.694+0000] {spark_submit.py:524} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2024-10-24T01:33:59.694+0000] {spark_submit.py:524} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2024-10-24T01:33:59.695+0000] {spark_submit.py:524} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2024-10-24T01:33:59.695+0000] {spark_submit.py:524} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-10-24T01:33:59.695+0000] {spark_submit.py:524} INFO - Caused by: java.util.concurrent.TimeoutException: Futures timed out after [120 seconds]
[2024-10-24T01:33:59.696+0000] {spark_submit.py:524} INFO - at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:259)
[2024-10-24T01:33:59.696+0000] {spark_submit.py:524} INFO - at scala.concurrent.impl.Promise$DefaultPromise.result(Promise.scala:263)
[2024-10-24T01:33:59.696+0000] {spark_submit.py:524} INFO - at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:48)
[2024-10-24T01:33:59.696+0000] {spark_submit.py:524} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2024-10-24T01:33:59.697+0000] {spark_submit.py:524} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2024-10-24T01:33:59.697+0000] {spark_submit.py:524} INFO - ... 8 more
[2024-10-24T01:33:22.915+0000] {job.py:219} ERROR - Job heartbeat got an exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
           ^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
          ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: could not translate host name "postgres" to address: Temporary failure in name resolution


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/jobs/job.py", line 215, in heartbeat
    heartbeat_callback(session)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/session.py", line 74, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/jobs/local_task_job_runner.py", line 246, in heartbeat_callback
    self.task_instance.refresh_from_db()
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/session.py", line 77, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 864, in refresh_from_db
    ti = qry.one_or_none()
         ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/query.py", line 2850, in one_or_none
    return self._iter().one_or_none()
           ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/query.py", line 2916, in _iter
    result = self.session.execute(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 1716, in execute
    conn = self._connection_for_bind(bind)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 1555, in _connection_for_bind
    return self._transaction._connection_for_bind(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 750, in _connection_for_bind
    conn = bind.connect()
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/future/engine.py", line 406, in connect
    return super(Engine, self).connect()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3325, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3404, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3374, in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 2208, in _handle_dbapi_exception_noconnection
    util.raise_(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
           ^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
          ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2024-10-24T01:33:59.735+0000] {spark_submit.py:524} INFO - 24/10/24 01:33:59 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 45953.
[2024-10-24T01:33:59.735+0000] {spark_submit.py:524} INFO - 24/10/24 01:33:59 INFO NettyBlockTransferService: Server created on de5108c6da1d:45953
[2024-10-24T01:33:59.754+0000] {spark_submit.py:524} INFO - 24/10/24 01:33:59 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-10-24T01:33:59.906+0000] {spark_submit.py:524} INFO - 24/10/24 01:33:59 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, de5108c6da1d, 45953, None)
[2024-10-24T01:34:00.007+0000] {spark_submit.py:524} INFO - 24/10/24 01:33:59 INFO BlockManagerMasterEndpoint: Registering block manager de5108c6da1d:45953 with 434.4 MiB RAM, BlockManagerId(driver, de5108c6da1d, 45953, None)
[2024-10-24T01:34:00.054+0000] {spark_submit.py:524} INFO - 24/10/24 01:34:00 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, de5108c6da1d, 45953, None)
[2024-10-24T01:34:00.095+0000] {spark_submit.py:524} INFO - 24/10/24 01:34:00 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, de5108c6da1d, 45953, None)
[2024-10-24T01:35:25.317+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to None. Terminating instance.
[2024-10-24T01:35:25.412+0000] {process_utils.py:131} INFO - Sending 15 to group 8858. PIDs of all processes in the group: [8920, 9593, 8858]
[2024-10-24T01:35:25.413+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 8858
[2024-10-24T01:35:25.414+0000] {taskinstance.py:1630} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-10-24T01:35:25.423+0000] {spark_submit.py:654} INFO - Sending kill signal to spark-submit
[2024-10-24T01:35:25.471+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 183, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 438, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 492, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 1632, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2024-10-24T01:35:25.490+0000] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=api_openbrewerydb_to_silver_gold, task_id=Silver, execution_date=20231202T080000, start_date=20241024T012658, end_date=20241024T013525
[2024-10-24T01:35:25.551+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 141 for task Silver (Task received SIGTERM signal; 8858)
[2024-10-24T01:35:25.788+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=8920, status='terminated', started='01:26:58') (8920) terminated with exit code None
[2024-10-24T01:35:25.789+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=8858, status='terminated', exitcode=1, started='01:26:57') (8858) terminated with exit code 1
[2024-10-24T01:35:25.790+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=9593, status='terminated', started='01:27:21') (9593) terminated with exit code None
[2024-10-24T14:35:32.783+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver permission to 509
[2024-10-24T14:35:32.784+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver'
[2024-10-24T14:35:32.811+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver permission to 509
[2024-10-24T14:35:32.811+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver'
[2024-10-24T14:35:32.840+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_openbrewerydb_to_silver_gold.Silver scheduled__2023-12-02T08:00:00+00:00 [queued]>
[2024-10-24T14:35:32.848+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_openbrewerydb_to_silver_gold.Silver scheduled__2023-12-02T08:00:00+00:00 [queued]>
[2024-10-24T14:35:32.848+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-10-24T14:35:32.857+0000] {taskinstance.py:1380} INFO - Executing <Task(SparkSubmitOperator): Silver> on 2023-12-02 08:00:00+00:00
[2024-10-24T14:35:32.862+0000] {standard_task_runner.py:57} INFO - Started process 13612 to run task
[2024-10-24T14:35:32.864+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'api_openbrewerydb_to_silver_gold', 'Silver', 'scheduled__2023-12-02T08:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/api_openbrewerydb_to_silver.py', '--cfg-path', '/tmp/tmpfse8gthg']
[2024-10-24T14:35:32.865+0000] {standard_task_runner.py:85} INFO - Job 89: Subtask Silver
[2024-10-24T14:35:32.912+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver permission to 509
[2024-10-24T14:35:32.912+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=api_openbrewerydb_to_silver_gold/run_id=scheduled__2023-12-02T08:00:00+00:00/task_id=Silver'
[2024-10-24T14:35:32.913+0000] {task_command.py:415} INFO - Running <TaskInstance: api_openbrewerydb_to_silver_gold.Silver scheduled__2023-12-02T08:00:00+00:00 [running]> on host 96f89615839b
[2024-10-24T14:35:32.992+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rodrigo Araujo' AIRFLOW_CTX_DAG_ID='api_openbrewerydb_to_silver_gold' AIRFLOW_CTX_TASK_ID='Silver' AIRFLOW_CTX_EXECUTION_DATE='2023-12-02T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-02T08:00:00+00:00'
[2024-10-24T14:35:33.003+0000] {base.py:73} INFO - Using connection ID 'spark-conn' for task execution.
[2024-10-24T14:35:33.003+0000] {spark_submit.py:363} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --name arrow-spark jobs/silver/api_openbrewerydb.py
[2024-10-24T14:35:35.763+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SparkContext: Running Spark version 3.5.3
[2024-10-24T14:35:35.765+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SparkContext: OS info Linux, 5.15.0-124-generic, amd64
[2024-10-24T14:35:35.771+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SparkContext: Java version 11.0.25
[2024-10-24T14:35:35.823+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-10-24T14:35:35.901+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO ResourceUtils: ==============================================================
[2024-10-24T14:35:35.901+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-10-24T14:35:35.901+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO ResourceUtils: ==============================================================
[2024-10-24T14:35:35.902+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SparkContext: Submitted application: IngestaoSilver
[2024-10-24T14:35:35.920+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-10-24T14:35:35.933+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO ResourceProfile: Limiting resource is cpu
[2024-10-24T14:35:35.933+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-10-24T14:35:35.977+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SecurityManager: Changing view acls to: ***
[2024-10-24T14:35:35.978+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SecurityManager: Changing modify acls to: ***
[2024-10-24T14:35:35.978+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SecurityManager: Changing view acls groups to:
[2024-10-24T14:35:35.978+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SecurityManager: Changing modify acls groups to:
[2024-10-24T14:35:35.979+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:35 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-10-24T14:35:36.190+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO Utils: Successfully started service 'sparkDriver' on port 34943.
[2024-10-24T14:35:36.213+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO SparkEnv: Registering MapOutputTracker
[2024-10-24T14:35:36.241+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO SparkEnv: Registering BlockManagerMaster
[2024-10-24T14:35:36.258+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-10-24T14:35:36.259+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-10-24T14:35:36.262+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-10-24T14:35:36.280+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-862f8b13-dfd5-438c-aefc-110636f16c2e
[2024-10-24T14:35:36.292+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-10-24T14:35:36.304+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-10-24T14:35:36.430+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-10-24T14:35:36.479+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-10-24T14:35:36.577+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-24T14:35:36.618+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO TransportClientFactory: Successfully created connection to spark-master/172.18.0.2:7077 after 24 ms (0 ms spent in bootstraps)
[2024-10-24T14:35:36.695+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20241024143536-0084
[2024-10-24T14:35:36.698+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20241024143536-0084/0 on worker-20241024142308-172.18.0.5-43899 (172.18.0.5:43899) with 2 core(s)
[2024-10-24T14:35:36.699+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO StandaloneSchedulerBackend: Granted executor ID app-20241024143536-0084/0 on hostPort 172.18.0.5:43899 with 2 core(s), 1024.0 MiB RAM
[2024-10-24T14:35:36.706+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 45307.
[2024-10-24T14:35:36.707+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO NettyBlockTransferService: Server created on 96f89615839b:45307
[2024-10-24T14:35:36.709+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-10-24T14:35:36.720+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 96f89615839b, 45307, None)
[2024-10-24T14:35:36.725+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO BlockManagerMasterEndpoint: Registering block manager 96f89615839b:45307 with 434.4 MiB RAM, BlockManagerId(driver, 96f89615839b, 45307, None)
[2024-10-24T14:35:36.728+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20241024143536-0084/0 is now RUNNING
[2024-10-24T14:35:36.728+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 96f89615839b, 45307, None)
[2024-10-24T14:35:36.730+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:36 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 96f89615839b, 45307, None)
[2024-10-24T14:35:37.008+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-10-24T14:35:37.262+0000] {spark_submit.py:524} INFO - Hello from Spark!
[2024-10-24T14:35:37.263+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-10-24T14:35:37.276+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO SparkUI: Stopped Spark web UI at http://96f89615839b:4040
[2024-10-24T14:35:37.283+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-10-24T14:35:37.288+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-10-24T14:35:37.308+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-10-24T14:35:37.318+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO MemoryStore: MemoryStore cleared
[2024-10-24T14:35:37.318+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO BlockManager: BlockManager stopped
[2024-10-24T14:35:37.325+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-10-24T14:35:37.326+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-10-24T14:35:37.336+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO SparkContext: Successfully stopped SparkContext
[2024-10-24T14:35:37.582+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO ShutdownHookManager: Shutdown hook called
[2024-10-24T14:35:37.583+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO ShutdownHookManager: Deleting directory /tmp/spark-ecc3b3d2-3b29-438a-a0ad-17b958a92f4d
[2024-10-24T14:35:37.586+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO ShutdownHookManager: Deleting directory /tmp/spark-ecc3b3d2-3b29-438a-a0ad-17b958a92f4d/pyspark-02cd8a43-5183-4b6a-b324-f13a8374d561
[2024-10-24T14:35:37.589+0000] {spark_submit.py:524} INFO - 24/10/24 14:35:37 INFO ShutdownHookManager: Deleting directory /tmp/spark-ac0bb387-8396-4f65-a2e9-6b38ebc46c9e
[2024-10-24T14:35:37.661+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=api_openbrewerydb_to_silver_gold, task_id=Silver, execution_date=20231202T080000, start_date=20241024T143532, end_date=20241024T143537
[2024-10-24T14:35:37.691+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-10-24T14:35:37.723+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
